src/retrace-server-task.txt

```diff
@@ -39,10 +39,14 @@ create::
 
 set::
     Write (set) information on an existing task. Available only for manager
-    and ftp task.
+    and ftp task. See retrace-server-interact for setting the success or
```

Comment:
The “See X. See Y.” bit reads to me as if Y can be found in X. I would put this last and maybe phrase it like “more information about setting the success and failure of the task can be found in the retrace-server-interact(1) manual page”.

Oh, and “Available only for manager and ftp task” needs some fixing.

Reply:
The confusion I had was that the additional options are further down in the same man page. I consider reorganizing the man page so that the respective options were listed immediately after "set" and "get", however I didn't want to get too crazy with the changes. 

I am not sure what fixes are needed for the “Available only for manager and ftp task”  line.

Reply:
The fixes needed are grammatical in nature. Using the plural “tasks” is what I have in mind specifically.

If you have an idea of what structural changes could make it better, knock yourself out, I’d say. I don’t think there are any good examples in the ABRT world, however (nor do I really know what makes a good man page).